plugins/chat/assets/stylesheets/common/chat-channel-title.scss

```
@@ -1,3 +1,4 @@
+//appears in: header of chat pane, channel info, preview card
```

```suggestion
// appears in: header of chat pane, channel info, preview card
```

Let's be civilized 😛

---

seriously? Im not restarting the checks just for this 🤣

---

Checks are just a suggestion 😛  For example red "x" by the system tests means "all good" 🤪